app/static/js/trackVolunteers.js

```diff
@@ -21,32 +21,50 @@ $(document).ready(function() {
       });
     });
 
-    // Adding the new volunteer to the user database table
-    $("#selectVolunteerButton").click(function(){
-      let user = $("#addVolunteerInput").val()
-      let eventId = $("#eventID").val()
-
-      $.ajax({
-        url: `/addVolunteerToEvent/${eventId}/${user}`,
-        type: "POST",
-        success: function(response){
-          console.log(response)
-          location.reload();
-        },
-        error: function(request, status, error){
-          location.reload();
-        }
-      });
-    });
 
-  function callback() {
-    $("#selectVolunteerButton").prop('disabled', false);
-  }
+  // Adding the new volunteer to the user database table
+    $("#selectVolunteerButton").click(function(){
+        let user = $("#addVolunteerInput").val()
+        let eventId = $("#eventID").val()
+        let checkboxlist = $("#addVolunteerModal input[type=checkbox]")
+        let volunteerList = []
+        $.each(checkboxlist, function(index, checkbox){
+            if(checkbox["checked"] == true){
+                volunteerList.push(checkbox["value"])
+            }
+        })
+        $.ajax({
+          url: `/addVolunteersToEvent/${eventId}`,
+          type: "POST",
+          data: {"volunteer" :volunteerList, "ajax": true},
+          success: function(s){
+              location.reload()
+          },
+          error: function(request, status, error){
+              location.reload()
+          }
+      })
+    })
 
+    var userlist = []
+    function callback(selected) {
+      $("#selectVolunteerButton").prop('disabled', false);
+      let user = $("#addVolunteerInput").val()
+      if(userlist.includes(selected["username"]) == false){
+          userlist.push(user)
+          let i = userlist.length;
+          $("#addVolunteerList").append("<li id= addVolunteerElements"+i+"> </li>")
```

Comment:
Now that we are using list items, we should style them so they look a little better. We don't need bullets.

Reply:
Let's also make sure that attribute value has quotes around it, even though it does work as is.